Client/src/Components/Sidebar/index.jsx

```diff
@@ -51,7 +52,14 @@ const menu = [
 	{ name: "Pagespeed", path: "pagespeed", icon: <PageSpeed /> },
 	{ name: "Infrastructure", path: "infrastructure", icon: <Integrations /> },
 	{ name: "Incidents", path: "incidents", icon: <Incidents /> },
-	// { name: "Status pages", path: "status", icon: <StatusPages /> },
+	{
+		name: "Status pages",
+		icon: <StatusPages />,
+		nested: [
+			{ name: "General Settings", path: "status/general-settings", icon: <Settings /> },
```

Comment:
This will open the "settings" submenu as it matches the path. 

```
useEffect(() => {
	const matchedKey = Object.keys(PATH_MAP).find((key) =>
		location.pathname.includes(key)
	);

	if (matchedKey) {
		setOpen((prev) => ({ ...prev, [PATH_MAP[matchedKey]]: true }));
	}
}, [location]);
```

This will have to be revised to function properly.

Reply:
I do not see what this part is doing, currently the nest menu item is collapsed initially, and when click on the "Status" on sidebar, the nested is expanded; Let's discuss in the meeting, I thought what we are trying to fulfil here is : 
1. When there is no status page settings, show the static page containing a create status page button
2. When there is status settings, hide the above static page; instead showing the status page with 2 nested panels, each has its entry in the sidebar
Is this correct?

Reply:
Sure let's discuss. I am not sure I understood this (couldn't wrap my head lol) but we can surely have a look during the meeting :)